Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:2
+
+WORKDIR /app
+
+ENV BASE_URL "https://www.mozilla.org"
+ENV BROWSER_NAME Firefox
+ENV BROWSER_VERSION 39.0
+ENV PLATFORM "Windows 7"
+ENV SAUCE_CREDENTIALS_PATH /tmp/creds
+ENV BUILD_TAG docker
```

Comment:
The build tag should be unique for each build, are we setting this somewhere else?